Modules/hpe3par_host.py

```diff
@@ -538,6 +539,20 @@ def add_fc_path_to_host(
             {})
     try:
         client_obj.login(storage_system_username, storage_system_password)
+
+        # check if wwn is already assigned
+        client_obj.setSSHOptions(storage_system_ip, storage_system_username, storage_system_password)
```

Comment:
Why are we setting ssh options?

Reply:
setSSHOptions is required for findHost() call

Reply:
SSH calls are heavy ops. Can we use queryHost() instead. It uses WSAPI call.